src/components/form/examples/custom-picker.tsx

```diff
@@ -7,21 +7,39 @@ import { ListItem } from 'src/components/list/list-item.types';
     shadow: true,
 })
 export class CustomPickerExample implements FormComponent<number> {
+    /**
+     * The value of the property
+     */
     @Prop()
     public value: number;
 
+    /**
+     * Label to display next to the input field
+     */
     @Prop()
-    public title: string;
+    public label: string;
```

Comment:
This is a breaking change!

Reply:
Oh, wait, no, this is just an example… 😄

Reply:
It's just in the example so it's fine :slightly_smiling_face: 
I think the example was not working before :smile:

Reply:
> I think the example was not working before 😄

You are absolutely correct! 😄